src/main/java/Entities/IngredientImpl.java

```diff
@@ -1,3 +1,5 @@
+/* Public class IngredientImpl that implements the Ingredient interface.
+ */
 package Entities;
```

Comment:
We do not need / want comments at the top of the file. If this is supposed to be the doc comment for the class, place it on the line preceding the class signature.